adapters/33across/usersync_test.go

```diff
@@ -4,13 +4,19 @@ import (
 	"testing"
 	"text/template"
 
+	"github.com/prebid/prebid-server/privacy"
 	"github.com/stretchr/testify/assert"
 )
 
 func Test33AcrossSyncer(t *testing.T) {
-	temp := template.Must(template.New("sync-template").Parse("https://ic.tynt.com/r/d?m=xch&rt=html&ri=123&ru=%2Fsetuid%3Fbidder%3D33across%26uid%3D33XUSERID33X&id=zzz000000000002zzz"))
```

Comment:
This code is a bit hard to read. Unified to an easier to read pattern. I originally had changes to these files, but that's been removed but I thought the refactor is nice to have.